src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_updateTier.java

```diff
@@ -38,14 +52,23 @@ public void createsTier() {
 
         List<ManagementTier> updatedTiers = managementTierRepository.findAll();
 
-        assertThat(updatedTiers.get(1).getId().getTier().getCodeValue()).isEqualTo("UB1");
-        assertThat(updatedTiers.get(1).getTierChangeReason().getCodeValue()).isEqualTo("ATS");
+        ManagementTier tierB1 = updatedTiers.get(1);
+        assertThat(tierB1.getId().getTier().getCodeValue()).isEqualTo("UB1");
+        assertThat(tierB1.getTierChangeReason().getCodeValue()).isEqualTo("ATS");
 
-    }
+        List<Contact> updatedContacts = contactService.contactsFor(2500343964L, contactFilter);
+        System.out.println("+++++++++++++");
```

Comment:
I guess you don't need this sout anymore?

Reply:
Done